Serilog.Sinks.Network.Test/UDPTests.cs

```diff
@@ -28,49 +28,49 @@ public void ConfigureTestLogger(ITextFormatter formatter = null)
             _logger = new LoggerConfiguration()

```

Comment:
And here too => I know it's not introduced in this PR but can this method be private?